internal/grpc/services/usershareprovider/usershareprovider.go

```diff
@@ -172,7 +172,7 @@ func (s *service) GetShare(ctx context.Context, req *collaboration.GetShareReque
 	if err != nil {
 		return &collaboration.GetShareResponse{
 			Status: status.NewInternal(ctx, err, "error getting share"),
-		}, nil
+		}, err
```

Comment:
I believe this was one of those "tricks" to convert an error to a proper response (but with `Internal Error`) that does not carry an `err` on top.
So returning `nil` ought to be correct, and ideally the `NewInternal` could be changed.